og.module

```diff
@@ -179,6 +195,11 @@ function og_entity_access(EntityInterface $entity, $operation, AccountInterface
 function og_entity_create_access(AccountInterface $account, array $context, $bundle) {
   $entity_type_id = $context['entity_type_id'];
 
+  if (!empty($context['skip_og_permission'])) {
+    // We've already been here or we want to skip the access check.
+    return;
+  }
```

Comment:
Why is this necessary? This seems brittle to me and a potential security problem. We should not allow to skip access checks unless there is a very good reason for it.

Reply:
Not sure about the code- but I think I remember the reason though. When you create a node, it still doesn't belong to a group. It _might_ belong, but if for example the audience field is optional, than that node could be a "global" one (i.e. does not belong to a group).

So we need to check if that node could be created outside of OG context. If it can, then we don't have the final say.

Reply:
It looks like this is intended to avoid an endless loop, because this is calling itself again later on.

This approach feels very strange to me. The code below calls the access check, passing exactly the same parameters, from inside the access check? What information could this possibly be retrieving that we don't already have at this point.